tests/Baseline/hilti.optimization.dead_store/opt.hlt

```diff
@@ -0,0 +1,23 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+module foo {
+
+import hilti;
+
+global uint<8> b;
+global uint<64> x1;
+global uint<64> x2;
+global uint<8> c1 = 0;
+global uint<8> c2;
+
+public function uint<8> fun() {
+    uint8(10);
```

Comment:
For now this leaves the RHS in place as I haven't build removal of dead code (needs detection/heuristic for whether code is pure).

Reply:
To observe what is happening this test only executes the `dead_store` pass. With full optimizations this statement would be gone.